osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs

```diff
@@ -76,7 +76,15 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            configSensitivity.BindValueChanged(val => localSensitivity.Value = val.NewValue, true);
+            configSensitivity.BindValueChanged(val =>
+            {
+                var disabled = localSensitivity.Disabled;
+
+                localSensitivity.Disabled = false;
+                localSensitivity.Value = val.NewValue;
+                localSensitivity.Disabled = disabled;
```

Comment:
Not sure about doing it like this, but this seems to be the only sensible way here. (not sure leasing would make sense, or even work)

Reply:
I think if we are going to be disabling these settings, it would work much better to disable the group they are in, rather than each bindable.

But I don't believe we have support for that currently. I think this is fine for now.